kars/0004-virtualized-accelerators/README.md

```diff
@@ -0,0 +1,52 @@
+# KAR-0004: Virtualized Accelerators
+
+## Description
+
+For accelerators that support virtualized accelerator technologies (e.g. vGPU), provide well-defined mechanisms for these to be exposed and managed, maintaining consistency with physical fractional GPUs. 
+
+Forward-looking: Once the accelerator supports virtualized accelerator technologies as part of DRA, then the platform should use the DRA mechanism.
+
+## Motivation
+
+AI/ML workloads on Kubernetes need virtualized accelerators because they allow GPU capacity to be flexibly carved out and delivered even in virtualized or multi-tenant environments where physical devices aren’t directly assignable. Maintaining consistency with physical fractional GPUs ensures that workloads, scheduling, policies, and user expectations remain the same across both physical and virtual backends, enabling portability and avoiding fragmented, vendor-specific resource models.
```

Comment:
I know this has merged already, but can you clarify what you mean here by "consistency with physical fractional GPUs"? Looking specifically at NVIDIA's vGPU technology, I'm not sure it would match this criteria very well (at least not in its standard mode of operation).

Reply:
"Consistency" in this context means that when a container or workload is scheduled onto a Kubernetes node with a virtualized accelerator, it should experience the same Kubernetes semantics as if it was scheduled onto a fractional piece of a physical GPU. The goal is not to enforce hardware consistency, but to ensure that, from the workload’s perspective in Kubernetes, the semantics (such as resource guarantees, accounting, and lifecycle) are equivalent. Feel free to suggest changes to the wording if you think it can be better conveyed.

Reply:
Addressing feedback in https://github.com/kubernetes-sigs/wg-ai-conformance/pull/39